database/callbacks_disable/beforeMigrate__disable_subscription.sql

```diff
@@ -0,0 +1,8 @@
+DO $$
+DECLARE
+  _sub TEXT := '${env}_${network}_alltables';
```

Comment:
Conceptually, could there be subscriptions we want to keep alive during a migration?
If not, this could probably be done a bit more generically by disabling and enabling every subscription we can find.
But please only follow this suggestion if you know this is not going to break anything as I lack familiarity with this. When in doubt let's leave it as is.